static/src/javascripts/projects/common/modules/commercial/membership-engagement-banner-parameters.js

```diff
@@ -74,10 +74,15 @@ const supporterEngagementCtaCopyJustOne = (location: string): string =>
         ? 'Support the Guardian from as little as $1.'
         : 'Support the Guardian from as little as £1.';
 
+const getSupporterLinkUrl = (location: string): string =>
+    location === 'GB'
+        ? 'https://support.theguardian.com?bundle=contribute'
```

Comment:
Should be 'https://support.theguardian.com/?bundle=contribute'

Reply:
Out of interest, why is that? Is it idiomatic or is there a deeper reason? I'll make the change, just curious as to why

Reply:
Actually, I'm a bit behind the URI specifications, either version is valid.  https://stackoverflow.com/questions/1617058/ok-to-skip-slash-before-query-string/42193734#42193734 sums it up.

Although if you put it into Chrome's address bar without the slash, it adds it for you.